documentation/anonymization.markdown

```diff
@@ -1,79 +1,75 @@
+* Resource titles: all titles are anonymized expect for those belonging to Filebucket, Package, Service, and Stage.
+* Parameter names: are never anonymized.
+* Parameter values: everything is anonymized except for the values for `provider`, `ensure`, `noop`, `loglevel`, `audit`, and `schedule`.
+* Report log messages: are always anonymized.
+* File names: are always anonymized.
+* File numbers: are left as they are.
+* Log messages: are always anonymized.
+* Metrics: metric names in the `time` category are anonymized as resource types.
 
 ### Profile: low
 
-This profile is aimed at hiding security information specifically, but leaving most of the data in its original state. By default most things are not anonymized except for:
+This profile is aimed at hiding security information specifically, but leaving most of the data in its original state. However, by default, the following categories are annonymized:
 
-* node name: is always anonymized
-* parameter values: we specifically anonymize any values and messages for any parameter name containing the strings: password, pwd, secret, key, private. Everything else is left alone.
-* log messages: are always anonymized
+* Node name: is always anonymized.
+* Parameter values: this profile will anonymize values and messages for any parameter name containing the strings `password`, `pwd`, `secret`, `key`, or `private`. Everything else is left alone.
+* Log messages: are always anonymized.
 
-Verifying your Anonymized Data
+Verifying your anonymized data
 -----
 
-While the tool itself tries to achieve the documented level of anonymization, this is your precious data and it is recommended you analyze it first before sharing it with another party to ensure all your requirements are met.
+After anonymizing data with the `puppetdb export` tool, we **strongly recommend** that you analyze the anonymized data before sharing it with another party to ensure that all sensitive data has been scrubbed.
 
-The best way to do this, is to untar the export file, and analyze the contents:
+Simply untar the export file and analyze the contents:
 
     $ tar -xzf my-puppetdb-anonymized-export.tar.gz
     $ cd puppetdb-bak
 
-Inside this directory there is a directory for each content type: `reports`, `catalogs` and each file inside represents a node (and a report instance for reports). The data is represented as pretty-formatted JSON so you can open these files up yourself and use tools such as `grep` to find any specific information you might have wanted to anonymize.
+Inside this directory there is a directory for each content type (such as `reports` and `catalogs`), and each file inside represents a node (and a report instance for reports). The data is represented as human-readable JSON. You can open these files and use tools such as `grep` to check the status of specific information you wished to anonymize.
```

Comment:
anonymization didn't work for facts when this was written, but it does now. This might be a good chance to change it to mention them all.

...there are directories for each content type (reports, catalogs, and facts) ...

FWIW content type isn't really a special term in PDB. Not sure if "there are directories for reports, catalogs, and facts" would be clearer. wished vs wish feels weird to me too but maybe it's a point of preference.